isolatable_hardwares.cpp

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Please expand the commit with the scope of this function

Reply:
> The isolate hardware may be FRU (Field Replaceable Unit) unit and
to get the EntityPath of that hardware from phal cec device tree
we may need to find out its immediate parent FRU

This is not clear?

Or 

> The isolate hardware may be FRU (Field Replaceable Unit) subunit and
to get the EntityPath of that subunit from phal cec device tree
we may need to find out its immediate parent FRU to locate the right subunit under the corresponding parent

How about above one?